### ISSUE TITLE
Auto refresh breaks when listening on all interfaces

### ISSUE THREAD

**Issue by benjamin-thomas** [slamdata/purescript-halogen-template#42]
Hello!

Trying a bit of PureScript and Halogen today :)

Launching this command, I can access the web server fine from another device on my local network.

```
npm run serve -- --host=0.0.0.0
```

However, auto-refreshing the page won't work because the websocket tries to connect to the wrong address.

I'm not sure who owns that code but maybe there's something you could do about it?

```
runtime-c0fb628751bc271e.js:94         
WebSocket connection to 'ws://0.0.0.0:1234/' failed: Error in connection establishment: net::ERR_ADDRESS_INVALID
```

Hacky workaround to grab my local IP:

```
npm run serve -- --host=$(hostname -I | awk '{print $1}')
```
But that breaks localhost unfortunately :(

A minor nuisance

**Comment by garyb** [comment]
I guess this is something to do with `parcel` since that's what is behind the `serve` command, but I don't know anything about really. I'd be a little surprised if it can't be configured to support this case.

For serving with auto-reloading in my own projects I've used `live-server` which has worked well enough for the most part.

**Comment by benjamin-thomas** [comment]
Ah it's a parcel bug indeed. And fixed in the latest [version 2.8.3](https://github.com/parcel-bundler/parcel/releases/tag/v2.8.3). See this PR: https://github.com/parcel-bundler/parcel/pull/7357

Closing this, sorry for the bother :)
